vmdb/app/models/vm_scan.rb

```diff
@@ -1,4 +1,17 @@
 class VmScan < Job
+
+  #
+  # TODO: until we get location/offset read capability for OpenStack
+  # image data, OpenStack fleecing is prone to timeout (based on image size).
+  # We adjust the queue timeout in server_smart_proxy.rb, but that's not enough,
+  # we also need to adjust the job timeout here.
+  #
+  DEFAULT_TIMEOUT = defined?(RSpec) ? 300 : 3000
```

Comment:
The tests have the value hard coded in many places, including messages. Adding this check was much simpler than making all the required mods to the tests.